src/Microsoft.Health.Dicom.Functions.Client.UnitTests/DicomAzureFunctionsHttpClientTests.cs

```diff
@@ -110,16 +124,16 @@ public async Task GivenSuccessfulResponse_WhenGettingStatus_ThenReturnStatus()
 @$"
 {{
   ""OperationId"": ""{id}"",
-  ""Type"": ""Reindex"",
-  ""CreatedTime"": ""{createdDateTime}"",
-  ""LastUpdatedTime"": ""{createdDateTime.AddMinutes(15)}"",
-  ""Status"": ""Running""
+  ""Type"": ""{OperationType.Reindex}"",
+  ""CreatedTime"": ""{createdDateTime:O}"",
+  ""LastUpdatedTime"": ""{createdDateTime.AddMinutes(15):O}"",
```

Comment:
what is this for? `:O`

Reply:
also is it normal to indent json like this?

Reply:
That is the [format specifier used by `DateTime`](https://docs.microsoft.com/en-us/dotnet/standard/base-types/standard-date-and-time-format-strings#table-of-format-specifiers) to indicate "Round Trip." .NET string interpolation uses the [following syntax](https://docs.microsoft.com/en-us/dotnet/csharp/language-reference/tokens/interpolated#structure-of-an-interpolated-string):

```
{<interpolationExpression>[,<alignment>][:<formatString>]}
```